seqr/utils/social_auth_pipeline.py

```diff
@@ -0,0 +1,18 @@
+import logging
+import requests
+
+from django.shortcuts import redirect
+
+from seqr.views.utils.terra_api_utils import _get_call_args
```

Comment:
you should never import protected methods in python (any method that starts with `_`). If this is a helper method that is needed outside the file where it is defined, rename it. Also, if this is intended to be used outside of the terra api file, you probably should name it something more descriptive, like `get_terra_call_args`

Reply:
Converted `_anvil_call()` to public function and named `anvil_call()` because `anvil_call()` has a higher level than `get_call_args()` has. Changed it call `anvil_call()` here.